### PR TITLE
Buffs Radiation and expedition suits

### PR DESCRIPTION
**Pull request by hex37**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
-"Buffs" Radiation and expedition suits, removing their slowdown.

-Tanks the melee and bullet resistance on the expedition suit to compensate for slowdown removal

-Removes both from the loadout menu

/:cl:

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/lists/suits.dm [hunk 1]
```diff
@@ -188,3 +188,4 @@
 	display_name = "expedition suit"
 	path = /obj/item/clothing/suit/space/anomaly
 	cost = 2 //Some good armor but slowdown
+*/
```
Comment (r4d6) [inline code/modules/client/preference_setup/loadout/lists/suits.dm:191]:
```suggestion
*/

```

File: code/modules/research/xenoarchaeology/tools/anomaly_suit.dm [hunk 2]
```diff
@@ -1,14 +1,15 @@
 /obj/item/clothing/suit/space/anomaly
 	name = "Expedition suit"
-	desc = "A pressure resistant expedition suit partially capable of insulating against exotic alien energies."
+	desc = "A pressure resistant expedition suit partially capable of insulating against exotic alien energies and a myriad of other things. Much like a waterbear, it has defenses against many forms of nonconventional death, at the expense of zero defense against dying in all the normal ways."
 	icon_state = "anomaly_suit"
 	item_state = "anomaly_suit"
-	armor_list = list(melee = 25, bullet = 10, energy = 50, bomb = 0, bio = 100, rad = 100)
+	slowdown = 0
```
Comment (Trilbyspaceclone) [inline code/modules/research/xenoarchaeology/tools/anomaly_suit.dm:6]: 0 slowdown for 40 energy space suit

File: code/modules/client/preference_setup/loadout/lists/suits.dm [hunk 3]
```diff
@@ -170,7 +170,7 @@
 /datum/gear/suit/akira
 	display_name = "red jacket"
 	path = /obj/item/clothing/suit/storage/akira
-
+/*
 /datum/gear/suit/radhat
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/suits.dm:174]: This ruins a few peoples character slots

Comment (cdb-is-not-good) [reply code/modules/client/preference_setup/loadout/lists/suits.dm:174]: Ah. Is there a cleaner way to deal with that than removing it completely?

Comment (hex37) [reply code/modules/client/preference_setup/loadout/lists/suits.dm:174]: we are testing rn to make sure the damage isn't too terrible

**Comment by Trilbyspaceclone** [comment]
after hex's test its no longer game braking

**Pull request merged**